fxgl/src/main/kotlin/com/almasb/fxgl/dsl/components/view/HealthBarViewComponent.kt

```diff
@@ -6,34 +6,16 @@
 
 package com.almasb.fxgl.dsl.components.view
 
-import com.almasb.fxgl.ui.Position
-import com.almasb.fxgl.ui.ProgressBar
 import javafx.beans.property.SimpleIntegerProperty
 import javafx.scene.paint.Color
 
 /**
  *
  * @author Almas Baimagambetov (almaslvl@gmail.com)
  */
-class HealthBarViewComponent(x: Double, y: Double) : ChildViewComponent(x, y, false) {
-
-    val hpBar = ProgressBar()
+class HealthBarViewComponent(x: Double, y: Double) : GenericBarViewComponent(x, y, SimpleIntegerProperty(2999), Color.GREEN) {
```

Comment:
I think we should just delete this file. Users can easily create something similar with `GenericBarViewComponent` directly